var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -973,6 +973,13 @@ def define_tpl(trilinos_name, spack_name, have_dep):
             # the MacOSX{nn.n}.sdk since that breaks at every xcode update
             options.append(define_tpl_enable("DLlib", False))
 
+            # math_errhandling is an 'extern int' and not a constexpr in MacOS
+            # headers and this is not accounted for directly in the STK code
+            # so we must disable floating point exception handling in that case
```

Comment:
Dang, I thought we took care of this in the stk code months ago...
@jccrean can you see this? (I may not know his github id but I'll point him to this in an email)

Reply:
The patch I tried didn't seem to work and I don't seen any recent changes to the code for this.